src/network/connection.h

```diff
@@ -732,8 +732,7 @@ class Peer {
 		virtual void reportRTT(float rtt) {};
 
 		void RTTStatistics(float rtt,
-							std::string profiler_id="",
-							unsigned int num_samples=1000);
+							std::string profiler_id = "");
```

Comment:
Too many indents. Only one (or two) additional ones (to the first function definition line) look good.